app/wrappers/wakes/facebook_metrics_wrapper.rb

```diff
@@ -22,7 +22,7 @@ def share_counts
           hash[r['id']] = r.fetch('share').fetch('share_count')
         rescue KeyError
           # Some urls can have no 'share' keys. This is generally in the case when the url isn't valid.
-          Rails.logger.warn "Share count not received for #{r['id']}. Possibly an invalid url?"
+          Rails.logger.warn "Share count not received for #{r['id']}. Response is: #{r}"
```

Comment:
@benhutton That's a good question. Here is the documentation: https://developers.facebook.com/docs/graph-api/reference/v2.5/url. What do you think?